packages/react-server-dom-vite/src/ReactFlightDOMClientNode.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {Thenable} from 'shared/ReactTypes.js';
+
+import type {Response} from 'react-client/src/ReactFlightClient';
+
+import type {Readable} from 'stream';
+
+import {
+  createResponse,
+  getRoot,
+  reportGlobalError,
+  processBinaryChunk,
+  close,
+} from 'react-client/src/ReactFlightClient';
+
+function noServerCall() {
+  throw new Error(
+    'Server Functions cannot be called during initial render. ' +
+      'This would create a fetch waterfall. Try to use a Server Component ' +
+      'to pass data to Client Components instead.',
+  );
+}
+
+export function createServerReference<A: Iterable<any>, T>(
+  id: any,
+  callServer: any,
+): (...A) => Promise<T> {
+  return noServerCall;
+}
+
+function createFromNodeStream<T>(
+  stream: Readable,
+  moduleMap: string,
```

Comment:
Is it string?
https://github.com/facebook/react/pull/26926/files#diff-5595e10b950f4a2cfcac117fc1c47e020366697a662dd897a7f399ddbab686a9R177 seems an object.